aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -666,11 +700,29 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 
 			"tags": tagsSchema(),
 
+			"server_side_encryption": {
+				Type:          schema.TypeList,
+				Optional:      true,
+				Computed:      true,
```

Comment:
We should avoid `Optional` and `Computed` attributes unless necessary. In this case we can implement: `DiffSuppressFunc: suppressMissingOptionalConfigurationBlock,` instead to allow drift detection and remove the `CustomizeDiff` 👍

Reply:
Done.